compiler/rustc_trait_selection/src/opaque_types.rs

```diff
@@ -38,13 +38,16 @@ pub struct OpaqueTypeDecl<'tcx> {
     /// then `substs` would be `['a, T]`.
     pub substs: SubstsRef<'tcx>,
 
-    /// The span of this particular definition of the opaque type.  So
+    /// The span of this particular definition of the opaque type. So
     /// for example:
     ///
     /// ```
     /// type Foo = impl Baz;
     /// fn bar() -> Foo {
-    ///             ^^^ This is the span we are looking for!
+    /// //          ^^^ This is the span we are looking for!
+    /// // ...
+    /// # unimplemented!()
+    /// }
```

Comment:
I personally don't think that this is worth it and would instead use `ignore` here.

This makes reading the doc comment noisier and we don't really test any behavior with this example